snuba/datasets/storages/errors.py

```diff
@@ -164,8 +164,8 @@
     replacer_processor=ErrorsReplacer(
         schema=schema,
         required_columns=required_columns,
-        tag_column_map={"tags": promoted_tag_columns, "contexts": {}},
-        promoted_tags={"tags": list(promoted_tag_columns.keys()), "contexts": []},
+        tag_column_map={"tags": {}, "contexts": {}},
+        promoted_tags={"tags": [], "contexts": []},
```

Comment:
Am I missing something or, when deleting a promoted tag, now we would not be setting the column to NULL ?

https://github.com/getsentry/snuba/blob/master/snuba/datasets/errors_replacer.py#L415-L416

the `promoted_tags` field is used to decide which columns to set to NULL when the tag is removed.

Reply:
No, you are right - I forgot about this. Thanks.